addon/test-waiter.ts

```diff
@@ -128,4 +132,10 @@ export default class TestWaiter<T = Token> implements ITestWaiter<T> {
   reset(): void {
     this.items.clear();
   }
+
+  private getCompletedOperations(token: T) {
+    return typeof token === 'number'
```

Comment:
I think this should actually be:

```suggestion
    let type = typeof token;
    
    return token === null || (type !== 'function' && type !== 'object');
```

Reply:
I don't think so, as that would break the functionality (this is expecting to return a map-like object).

It looks like you're proposing to just invert the condition, to make it anything that's _not_ any of the above conditions, correct?

Reply:
I've updated it to look for everything but `function` and `object`.